README.md

```diff
@@ -77,6 +77,7 @@ This plugin is GDPR complient if you enable the deletion of remote objects.
 
 | Moodle version    | Totara version           | Branch                                                                                       | PHP  | MySQL   | PostgreSQL  |
 |-------------------|--------------------------|----------------------------------------------------------------------------------------------|------|---------|-------------|
+| Moodle 4.4+       |                          | [MOODLE_404_STABLE](https://github.com/catalyst/moodle-tool_objectfs/tree/MOODLE_402_STABLE) | 8.1+ | 8.0+    | 13+         |
```

Comment:
Need to update the link as well

Reply:
Also can you rebase the `MOODLE_404_STABLE` branch - has 1 commit extra from this morning

Reply:
Hi @matthewhilton ,

I have rebased and changed the branch link.